edenia/antelope-tools
# Fix/ath is not working 1229

### Fix TPS All Time High

### What does this PR do?

- Resolve #1229
- Resolve #1225 
- Fix the cpu usage percentage calculation
- Add stats of average cpu and net usage in the last hour, day and week
- Temporarily disable demux when missed blocks are disabled

### Steps to test

1. Run the project locally with the block history enable
2. Go to the main page or the stress test page
3. Check ATH and new cards related to resource utilization.

### Screenshot

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/ab6831dd-f1f0-4404-9696-77d1cd72f140)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/df40d21b-7485-4c63-af34-85b694d0c7eb)
